content/docs/apps/tic.md

```diff
@@ -0,0 +1,14 @@
+---
+menu:
+  docs:
+    parent: advanced
```

Comment:
I think we need an Org Manager section of some kind ("Managing orgs"?), and to put this page in that section instead of in the "Managing apps" section.